pytest-wdl-tests/tests/literals/test_data.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do you need this if the file is empty?

Reply:
> FileNotFoundError: Could not find any of test_data.json,tests/test_data.json starting from /Users/jggatter/Desktop/Projects/Testathon-2020/pytest-wdl-tests/tests/whitespace

I deleted the jsons and the input dictionaries from every line in the python script and it gave me that error message. Any way around supplying inputs to py-test wdl?